src/test-utils/dom/table/index.ts

```diff
@@ -132,4 +133,26 @@ export default class TableWrapper extends ComponentWrapper {
   findPagination(): PaginationWrapper | null {
     return this.findComponent(`.${styles['tools-pagination']}`, PaginationWrapper);
   }
+
+  findEditingCell(): ElementWrapper | null {
+    return this.findNativeTable().findByClassName(bodyCellStyles['body-cell-edit-active']);
+  }
+
+  findEditingCellInputSlot(): ElementWrapper | null {
```

Comment:
This should not be a public util.

When testing this interaction, you could do the following:

```
tableWrapper.findEditingCell().findSelect(); // or findInput() or whatever you have inside
```

This selector is only for your internal assertions. You can write this selector inline in the tests which needed it: https://github.com/cloudscape-design/components/search?q=useBrowser+selectors.js&type=code

No need to make it public